src/pages/HomePage.tsx

```diff
@@ -1,39 +1,59 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
 
-import logo from '../logo.svg';
+import logo from "../logo.svg";
 
-import SessionInfo from '../types.js'
+import SessionInfo from "../types.js";
 
 class HomePage extends Component {
-    readonly state: {sessionInfo: SessionInfo | null} = {sessionInfo: null};
-
-    componentDidMount() {
-        this.fetchSession()
-    }
-
-    fetchSession() {
-        fetch('/session')
-            .then((response) => response.json())
-            .then((data: {sessionInfo: SessionInfo}) => {
-              this.setState({sessionInfo: data});
-            });
-    }
-
-    render() {
-        return (
-        <div className="App">
-            <header className="App-header">
-            <img src={logo} className="App-logo" alt="logo" />
-            <h1>Kvasir Movies</h1>
-            <p>
-                {this.state.sessionInfo && this.state.sessionInfo.is_session_active ? `Welcome back, ${this.state.sessionInfo.email}! ` : ''}
-                Find 🎬 with 👫 :D
-            </p>
-            <a href='/login'>Log In</a>
-            </header>
-        </div>
-        );
-    }
+  readonly state: { sessionInfo: SessionInfo | null } = { sessionInfo: null };
```

Comment:
Did this whole thing get unindented by two spaces? Is this a lint thing?

Reply:
Yep, my editor was displaying 2 spaces but was committing 4 spaces for some reason; `prettier` fixed that